saltstack/base/salt/telemetry/init.sls

```diff
@@ -1,2 +1,33 @@
-include:
-  - {{ slspath }}.centos
\ No newline at end of file
+{% set platform = salt['environ.get']('OS_TYPE') %}
+{% if salt['environ.get']('ARCHITECTURE') == 'arm64' %}
+  {% set platform = platform ~ 'arm64' %}
+{% endif %}
+
+add_cdp_infra_tools_repo:
+  file.managed:
+    - name: /etc/yum.repos.d/cdp-infra-tools.repo
+    - source: salt://telemetry/yum/cdp-infra-tools.repo.j2
+    - template: jinja
+    - platform: "{{ platform }}"
+
+list_available_packages_from_cdp_infra_tools_repo:
+  cmd.run:
+    - name: yum --disablerepo="*" --enablerepo=cdp-infra-tools list available
+
+install_cdp_infra_tools_packages:
+  pkg.installed:
+    - pkgs:
+{% if salt['environ.get']('INCLUDE_CDP_TELEMETRY') == "Yes" %}
+      - cdp-telemetry
+{% endif %}
+{% if salt['environ.get']('INCLUDE_FLUENT') == "Yes" %}
+      - redhat-lsb-core # this will install redhat-lsb-core is required for fluent
```

Comment:
This line is redundant, redhat-lsb-core is always installed much earlier as it is needed by other stuff too.

Reply:
Nevermind, actually. Seems like fluent always needs this while the other components only need it on RHEL 8.

Reply:
https://github.com/hortonworks/cloudbreak-images/blob/86cca4e9571714bc8a0306ff6c1b71671d7e6a0c/scripts/salt-install.sh#L44

Reply:
the earlier install is for rhel8 only